### PR TITLE
3 improvements

### PR DESCRIPTION
**Pull request by encukou**
Best reviewed separately (but if there are no issues, let's let Travis handle them at once):
* Show non-Python dependents in package pages
* Adjust status descriptions that assumed this only is for Python 3 porting
* Move CSS to a separate file

### CODE REVIEW

File: portingdb/load_data.py [hunk 1]
```diff
@@ -297,3 +297,13 @@ def load_from_directories(data, directories):
...
+
+    # Consolidate non-Python requirers
+    for name, package in packages.items():
+        nonpy_requirers = {}
+        package.setdefault('needs_py2_command', False)
+        for rpm in package['rpms'].values():
+            for kind, names in rpm.get('non_python_requirers', {}).items():
+               if names:
+                    nonpy_requirers.setdefault(kind, set()).update(names)
+        package['non_python_requirers'] = nonpy_requirers
```
Comment (hroncok) [inline portingdb/load_data.py]: I've read this code 5 times and I have no idea what does it actually do. Sorry. I must miss some context.

**Comment by hroncok** [comment]
Consider the *Adjust status descriptions that assumed this only is for Python 3 porting* and *Move CSS to a separate file* commits approved.

File: portingdb/load_data.py (hunk 1 above)
Comment (hroncok) [reply portingdb/load_data.py]: Oh, so we already track this in

https://github.com/fedora-python/portingdb/blob/4bce049022122493e4ce813bf20cf2224a85e0ad/dnf-plugins/py3query.py#L353-L356

File: portingdb/load_data.py [hunk 2]
```diff
@@ -297,3 +297,13 @@ def load_from_directories(data, directories):
             package['ftbfs_age'] = max(CURRENT_FEDORA - releasever, 0)
         else:
             package['ftbfs_age'] = 0
+
+    # Consolidate non-Python requirers
+    for name, package in packages.items():
+        nonpy_requirers = {}
+        package.setdefault('needs_py2_command', False)
```
Comment (hroncok) [inline portingdb/load_data.py]: I'Ve tried to remove this line and it seems the website works just fine without. Will check if there are any differences.

File: portingdb/load_data.py (hunk 1 above)
Comment (encukou) [reply portingdb/load_data.py]: Yes, but that's per RPM.
We probably don't need it per-RPM, but I'd like to keep old `fedora.json` formats loadable, so it's probably better to postprocess in load_data.

File: portingdb/load_data.py (hunk 2 above)
Comment (encukou) [reply portingdb/load_data.py]: You're right, that's a merge artifact.

Comment (hroncok) [inline portingdb/load_data.py]:
```suggestion
```

**Comment by hroncok** [comment]
As a followup readability improvement, maybe we can link pagure and spec with icons?

https://fontawesome.com/icons/code-branch?style=solid
https://fontawesome.com/icons/scroll?style=solid

**Comment by encukou** [comment]
Would be that useful often? My first attempt at it looked less readable – too cluttered.
Note that portingdb uses Font Awesome 4, the previous version.

**Pull request merged**